subworkflows/merge_tables_wf.nf

```diff
@@ -0,0 +1,41 @@
+/*
+ * Check input samplesheet and get read channels
+ */
+
+params.options = [:]
+
+include { MERGE_TABLES } from '../modules/local/merge_tables' addParams( options: params.options )
+
+workflow MERGE_TABLES_WF {
+    take:
+    tables // file: /path/to/samplesheet.tsv
+    
+    main:
+    // TODO: avoiding checking samplesheet for now, add samplesheet check later.
```

Comment:
copy & paste from the other subworkflow?

Reply:
yup 😄